reactor-netty-http/src/test/java/reactor/netty/http/client/HttpClientTest.java

```diff
@@ -2282,6 +2283,15 @@ public void testTcpConfigurationUnsupported_4() {
 		                                    }));
 	}
 
+	@Test
+	public void testApplyTcpClientSSLConfig() throws Exception {
```

Comment:
nit: Remove `public`. We are using JUnit5 and this is not needed anymore.

Reply:
Removed 👍